### ISSUE TITLE
Proxyuser for Hive View

### ISSUE THREAD

**Issue by ro-ket** [Isilon/isilon_hadoop_tools#11]
When I added proxyuser creation recently, I forgot to include one needed for Hive View (and perhaps other Ambari Views):

isi hdfs proxyusers create --proxyuser root --add-group hadoop
isi hdfs proxyusers modify --proxyuser root --add-group Users

It's included in the description of Hive View configuration in this DECN blog post, https://community.emc.com/community/products/isilon/blog/2016/08/16/configuring-ambari-hive-view-with-onefs

This is not necessary for Cloudera clusters.

**Comment by bonibruno** [comment]
Note:  CLI syntax on the Proxy Users section above is incorrect, it's missing the equal sign to assign the value to each option listed.  Also, since access zones are typically used in production deployments, the --zone option will likely be needed.  The correct syntax for reference is:
 
isi hdfs proxyusers create --proxyuser=root --add-group=hadoop --zone={your zone name}
isi hdfs proxyusers modify --proxyuser=root --add-group=Users --zone={your zone name}

**Comment by dmtucker** [comment]
Looks like [`--zone` is already in place](https://github.com/Isilon/isilon_hadoop_tools/pull/10/files#diff-07e50b308d76ed7ccb6ab0d4b5b4f209R266).
Also, that `=` is actually optional:

```
dtucker-onefs-1# isi hdfs proxyusers list -v                                                 
dtucker-onefs-1# isi hdfs proxyusers create --proxyuser root --add-group Users --zone System
dtucker-onefs-1# isi hdfs proxyusers list -v                                                
   Name: root
Members: Users

**Comment by bonibruno** [comment]
If you use the default System zone then you can ignore the zone flag, however most production Hadoop deployments do create a separate access zone for HDFS, so if a separate zone was created, make sure you stipulate the zone flag.  :)

**Comment by ro-ket** [comment]
@bonibruno , this issue still needs to be fixed. It wasn't included in #10. I can't reopen it, but I'll submit a PR to take care of it tomorrow.

I think @dmtucker meant that the code you merged today already includes the zone flag. 

We need to add the proxy user `root` but only in the `hwx` block of the case statement. It needs to include both the `hadoop` group and the smokeuser. (This is what I meant when I opened the issue with hasty copy and paste.)
